utils_cv/tracking/model.py

```diff
@@ -39,48 +38,6 @@
 from .plot import draw_boxes, assign_colors
 from ..common.gpu import torch_device
 
-BASELINE_URL = (
-    "https://drive.google.com/open?id=1udpOPum8fJdoEQm6n0jsIgMMViOMFinu"
-)
-
-
-def _download_baseline(url, destination) -> None:
```

Comment:
Is this function not needed anymore?

Reply:
It never really worked lol because I couldn't find the url to directly download